builder_data.yaml

```diff
@@ -50,10 +50,11 @@ apt_repo_command: |
   keychain -k mine;
 yum_repo_command: |
   for repodir in $(find "__REPO_PATH__" -mindepth 2 -name "*.rpm" | xargs -I {} dirname {} | uniq) ; do
-    [ -d "${repodir}" ] || continue;
-    echo "Generating repodata for ${repodir}..."
-    sudo createrepo --checksum=sha --database --update "${repodir}";
-    echo "Finished generating repodata for ${repodir}..."
+    [ -d "${repodir}" ] || continue &&
+    echo "Generating and signing repodata for ${repodir} with __GPG_KEY__..." &&
+    sudo createrepo --checksum=sha --database --update "${repodir}" &&
+    gpg --armor --detach-sign -u __GPG_KEY__ "${repodir}/repodata/repomd.xml" &&
+    gpg --export --armour __GPG_KEY__ > "${repodir}/repodata/repomd.xml.key"
```

Comment:
Alright, this does work. The issue lies with subkeys. Sles 11 panics if there are subkeys attached to a key, and cannot verify the package signature. If everyone running this command doesn't have the subkeys, then we're fine. But that's unrealistic. I need to find a way to export the key, but ignore any subkeys.